src/anomalib/engine/engine.py

```diff
@@ -161,12 +164,63 @@ def trainer(self) -> Trainer:
             raise UnassignedError(msg)
         return self._trainer
 
+    @property
+    def model(self) -> AnomalyModule:
+        """Property to get the model.
+
+        Raises:
+            UnassignedError: When the model is not assigned yet.
+
+        Returns:
+            AnomalyModule: Anomaly model.
+        """
+        if not self.trainer.model:
+            msg = "Trainer does not have a model assigned yet."
+            raise UnassignedError(msg)
+        return self.trainer.lightning_module
+
+    @property
+    def normalization_callback(self) -> NormalizationCallback | None:
+        """The ``NormalizationCallback`` callback in the trainer.callbacks list, or ``None`` if it doesn't exist.
+
+        Returns:
+            Optional[_MinMaxNormalizationCallback]: Normalization callback, if available.
+
+        Raises:
+            ValueError: If there are multiple normalization callbacks.
+        """
+        callbacks = [callback for callback in self.trainer.callbacks if isinstance(callback, NormalizationCallback)]
+        if len(callbacks) > 1:
+            msg = "Trainer can only have one normalization callback but multiple found: {}. Please check your \
+                configuration. Exiting to avoid unexpected behavior.".format(callbacks)
+            raise ValueError(msg)
+        return callbacks[0] if len(callbacks) > 0 else None
+
+    @property
+    def threshold_callback(self) -> _ThresholdCallback | None:
+        """The ``ThresholdCallback`` callback in the trainer.callbacks list, or ``None`` if it doesn't exist.
+
+        Returns:
+            Optional[_ThresholdCallback]: Threshold callback, if available.
+
+        Raises:
+            ValueError: If there are multiple threshold callbacks.
+        """
+        callbacks = [callback for callback in self.trainer.callbacks if isinstance(callback, _ThresholdCallback)]
+        if len(callbacks) > 1:
+            msg = f"Trainer can only have one thresholding callback but multiple found: {callbacks}. Please check your \
+                configuration. Exiting to avoid unexpected behavior."
```

Comment:
```suggestion
            msg = (
                f"Trainer can only have one thresholding callback but multiple found: {callbacks}. "
                "Please check your configuration. Exiting to avoid unexpected behavior."
            )
```

Reply:
same as above